docs/source/advanced_topics/0_dead_stream_eviction.rst

```diff
@@ -0,0 +1,27 @@
+Dead Streams Eviction
+=====================
+
+Savant automatically evicts dead streams. You know about such evictions by receiving corresponding events (via designated callbacks) in your Pyfuncs: normally, you do not need to deal with evictions somehow. Instead, configuration parameters specified per source tune how frequently Savant checks the idle streams and how long the stream can be inactive before being considered dead.
```

Comment:
` normally, you do not need to deal with evictions somehow` -> `normally, you do not need to manually deal with evictions`